pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.devonfw.tools</groupId>
 	<artifactId>solicitor</artifactId>
-	<version>1.2.0-SNAPSHOT</version>
```

Comment:
Please do not change version number here. Rebase you branch on the latest master instead.